website/pages/consul-on-kubernetes/index.tsx

```diff
@@ -231,6 +232,38 @@ export default function ConsulOnKubernetesPage() {
       </section>
 
       {/* get started section */}
+      <section>
+        {/* card list */}
+        <DocsList
+          title="Documentation"
+          docs={[
+            {
+              icon: {
+                src: require('./images/docs/helm-icon.png'),
```

Comment:
Did some digging & found a SVG for this, mind swapping to that ✨ 

https://cncf-branding.netlify.app/projects/helm/